data/annulus-pi-3.msh

```diff
@@ -0,0 +1,236 @@
+$MeshFormat
```

Comment:
This will be the first time we add Gmsh files in the repository. Do we want to do that, or is https://github.com/mfem/data/tree/master/gmsh a better place for them?